README.md

```diff
@@ -10,6 +10,9 @@ The current version of the proxy utilizes Google Cloud Pub/Sub as the backend Pu
 If you're planning on running multiple instances of the proxy for scalability,
 you must avoid running multiple instances on the same host.
 
+If you will be using Google Cloud Pub/Sub, you must set the GCLOUD_PUBSUB_PROJECT_ID environment
```

Comment:
Why not changing it now? It is confusing if we change the environment variable name.